File: app/src/main/java/se/kicksort/metalarchives/model/Song.java
package se.kicksort.metalarchives.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

/**
 * Created by Gustav Sundin on 17/03/17.
 */

public class Song {
    @SerializedName("title")
    @Expose
    private String title;

    public String getTitle() {
        return title;
    }

    //TODO
    public String getLength() {
        return "";
    }
}

File: app/src/main/java/se/kicksort/metalarchives/model/Song.kt
package se.kicksort.metalarchives.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

/**
 * Created by Gustav Sundin on 17/03/17.
 */

class Song {
    @SerializedName("title")
    @Expose
    val title: String? = null

    //TODO
    val length: String
        get() = ""
}
